src/ClientInterface.php

```diff
@@ -89,6 +91,8 @@
  * @method int               decr(string $key)
  * @method int               decrby(string $key, int $decrement)
  * @method Status            failover(?To $to = null, bool $abort = false, int $timeout = -1)
+ * @method Status            ftcreate(string $index, CreateArguments $arguments, Schema $schema)
+ * @method array             ftsearch(string $index, string $query, CreateArguments $arguments)
```

Comment:
`CreateArguments $arguments`?!

Reply:
It was `Arguments` first, but then @sazzad16 suggest to rename it to `CreateArguments`, for me it more feels like `SearchArgument`, this object is all about creating an array of arguments

Reply:
to me `CreateArguments` suggests it's the arguments for `FT.CREATE`... I would rename it `SearchArguments`

Reply:
@leibale Fixed